src/components/core/tabs/stories/tabs.stories.tsx

```diff
@@ -41,37 +35,74 @@ export default {
                 defaultValue: { summary: 'default' },
             },
         },
+        default_selected: {
+            defaultValue: 'Deposit',
+        },
     },
 };
 
 const Template: Story<TabsProps> = (args, { globals: { theme } }) => {
     const isDark = theme === 'dark';
+    const tabs = [
+        {
+            label: 'Deposit',
+            icon: isDark ? DepositIconDark : DepositIconLight,
+            content: 'Deposit',
+        },
+        {
+            label: 'Withdraw',
+            icon: isDark ? WithdrawIconDark : WithdrawIconLight,
+            content: 'Withdraw',
+        },
+        {
+            label: 'Transfer',
+            icon: isDark ? TransferIconDark : TransferIconLight,
+            content: 'Transfer',
+        },
+        {
+            label: 'Transactions',
+            icon: isDark ? TransactionIconDark : TransactionIconLight,
+            content: 'Transactions',
+        },
+    ];
     return (
         <Tabs {...args}>
-            <Tab icon={isDark ? DepositIconDark : DepositIconLight} label="Deposit">
-                Deposit
-            </Tab>
-            <Tab icon={isDark ? WithdrawIconDark : WithdrawIconLight} label="Withdraw">
-                Withdraw
-            </Tab>
-            <Tab icon={isDark ? TransferIconDark : TransferIconLight} label="Transfer">
-                Transfer
-            </Tab>
-            <Tab icon={isDark ? TransactionIconDark : TransactionIconLight} label="Transactions">
-                Transactions
-            </Tab>
+            <Tabs.List {...args}>
+                <Tabs.Trigger value="Deposit" icon={isDark ? DepositIconDark : DepositIconLight}>
+                    Deposit
+                </Tabs.Trigger>
+                <Tabs.Trigger value="Withdraw" icon={isDark ? WithdrawIconDark : WithdrawIconLight}>
+                    Withdraw
+                </Tabs.Trigger>
+                <Tabs.Trigger value="Transfer" icon={isDark ? TransferIconDark : TransferIconLight}>
+                    Transfer
+                </Tabs.Trigger>
+                <Tabs.Trigger value="Transactions" icon={isDark ? TransactionIconDark : TransactionIconLight}>
+                    Transactions
+                </Tabs.Trigger>
+            </Tabs.List>
+            <TabsContent value="Deposit">
+                <p>Deposit</p>
+            </TabsContent>
+            <TabsContent value="Withdraw">
+                <p>Withdraw</p>
+            </TabsContent>
+            <TabsContent value="Transfer">
+                <p>Transfer</p>
+            </TabsContent>
+            <TabsContent value="Transactions">
+                <p>Transactions</p>
+            </TabsContent>
```

Comment:
Since we already declared the `tabs` array, can we just iterate through it instead of doing it one by one like this? Much cleaner. Apart from that everything is good 👍🏼